### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by laurapacilio**
### What
We're shipping versioned documentation for Cloud Agents. This PR adds context to the README about deployment, to help folks understand when their changes will be live on the site. This is to match work we've done in other repositories.

### Why
Help contributors understand how to make their content live/how the site generates versioned documentation for Cloud Agents.

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [ ] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

**Comment by laurapacilio** [comment]
@thiskevinwang - is this correct for how the Cloud Agents process is working?

File: README.md [hunk 1]
```diff
@@ -317,4 +321,4 @@ In markdown, images should be referenced by their absolute path, starting with `
 
 ## Redirects
 
-For now, redirects will need to be handled here: https://github.com/hashicorp/terraform-website/blob/master/redirects.next.js
\ No newline at end of file
+For now, redirects will need to be handled here: https://github.com/hashicorp/terraform-website/blob/master/redirects.next.js
```
Comment (laurapacilio) [inline README.md:324]:
```suggestion
You must add a redirect when you move, rename, or delete documentation pages. Refer to https://github.com/hashicorp/terraform-website#redirects for details.
```

File: README.md [hunk 2]
```diff
@@ -63,25 +63,29 @@ If you find a typo or you feel like you can improve the HTML, CSS, or JavaScript
...
+**Set Up Local Environment**
+
+1. [Install Docker](https://docs.docker.com/get-docker/).
+1. Restart your terminal or command line session.
+
+**Launch Site Locally**
+
+1. Navigate into your local `terraform-plugin-mux` top-level directory and run `make website`. 
+1. Open `http://localhost:3000` in your web browser. While the preview is running, you can edit pages and Next.js will automatically rebuild them.
+1. When you're done with the preview, press `ctrl-C` in your terminal to stop the server.
```
Comment (thiskevinwang) [inline README.md:81]: I think we can retain the previous sections (`## Running the Site Locally` and `### Using Docker`) as they are still accurate.

The `## Deployment` section is worth keeping though!

File: README.md [hunk 3]
```diff
@@ -63,25 +63,29 @@ If you find a typo or you feel like you can improve the HTML, CSS, or JavaScript
...
-- `make website/build-local`
-  - This command will build a local image of the website from `hashicorp/dev-portal.git`.
+**Set Up Local Environment**
+
+1. [Install Docker](https://docs.docker.com/get-docker/).
+1. Restart your terminal or command line session.
+
+**Launch Site Locally**
+
+1. Navigate into your local `terraform-plugin-mux` top-level directory and run `make website`.
```
Comment (thiskevinwang) [inline README.md:79]:
```suggestion
1. Navigate into your local `terraform-docs-common` top-level directory and run `make website`. 
```

**Review by thiskevinwang — APPROVED** [review]
1 typo, otherwise ✅

**Review by ryanuber — APPROVED** [review]
Seems good to me!

File: README.md (hunk 3 above)
Comment (laurapacilio) [reply README.md:79]: oh wait!! this isn't even common - it's this directory - thank you!

**Pull request merged**